x-pack/plugins/cases/public/components/case_action_bar/index.tsx

```diff
@@ -40,6 +40,7 @@ interface CaseActionBarProps {
   caseData: Case;
   currentExternalIncident: CaseService | null;
   disabled?: boolean;
+  isAlerting: boolean;
```

Comment:
nit: `disableAlerts` seems a better name for this variable.

Reply:
its confusing to me because 42 `disabled` property is for alerts

Reply:
depending on userCanCrud

Reply:
oh I might have added that 😆  feel free to rename it, it was just trying to fix an issue where the status button was enabled when the user only had read only permissions.